apps/frontend/src/app/5_pages/ZeroPage/ZeroPage.tsx

```diff
@@ -59,10 +60,13 @@ export const ZeroPage: FC = () => {
   const { refetch: getOpenTroves } = useGetUserOpenTrove(account);
 
   const collateral = useMemo(
-    () => Number(trove?.collateral ?? 0),
+    () => Decimal.from(trove?.collateral?.toString() || 0),
```

Comment:
We have this type of condition (`Decimal.from(someValue || 0)` or `Decimal.from(someValue ?? 0)) in multiple places. What do you think about adjusting `numeric` to accept `Decimalish | undefined | null` and returning `Decimal.ZERO` in case it's `undefined` or `null`? You could then use `numeric` on those places and I think it would be much easier to use in the future + the codebase would be cleaner.

Reply:
Yep, it makes sense